package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "description": "An enterprise-class UI components based on Ant Design and Angular",
   "engines": {
-    "node": "^12.20.0 || ^14.0.0"
+    "node": "^12.20.0 || ^14.0.0 || ^16.0.0"
```

Comment:
Perhaps follow the Angular engine node version ```"node": "^12.20.0 || ^14.15.0 || >=16.10.0" ```

Reply:
The caret sign already implies a wider version range, which should be sufficient.